app/views/people/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= render "shared/header" %>
+
+  <main>
+ <table class="details">
+  <tr>
+    <th>Id</th>
+    <td><%= @person.id %></td>
+  </tr>
+  <tr>
+    <th>Name</th>
+    <td><%= @person.name %></td>
+  </tr>
+</table>
+<section class="button_placement">
```

Comment:
Good to remember. I have been bad at implementing CSS classes in past projects, so I think I've been a bit squidgey on the conventions.